src/pages/user-info/user-info.css

```diff
@@ -0,0 +1,9 @@
+.password-field {
+    position: relative !important;
+}
+
+.show-password-icon {
+    position: absolute !important;
+    top: 35px !important;
+    right: 10px !important;
```

Comment:
- I think you are using `!important` too often, maybe try to think some other way of overriding the styles.